src/main.ts

```diff
@@ -95,12 +95,53 @@ export function isSuccess<E, A>(monadA: Dataway<E, A>): monadA is Success<A> {
   return monadA._tag === 'Success';
 }
 
+/**
+ * Construct a new `Dataway` from a nullable type, if the value is `null` or `undefined`,
+ * returns `NotAsked` else returns the value wrapped in a `Success`
+ *
+ * | parameter    | Result        |
+ * | ------------ | ------------- |
+ * | null         | NotAsked      |
+ * | undefined    | NotAsked      |
+ * | foo          | Success foo   |
+ *
+ *
+ * ```
+ * import { notAsked, success, fromNullable } from  'dataway';
+ *
+ * fromNullable(null) === notAsked;
+ * fromNullable('foo') === success('foo');
+ * ```
+ * @typeparam A type of the provided value
+ */
 export function fromNullable<E = never, A = never>(
   successValue: A | null | undefined,
 ): Dataway<E, A> {
   return successValue == null ? notAsked : success(successValue);
 }
 
+/**
+ * Construct a new `Dataway` from a `fp-ts` `Either` type, if the value is a `Left value`,
+ * returns `Failure value` else returns the `Right` value wrapped in a `Success`
```

Comment:
```
Constructs a new `Dataway` from a `fp-ts` `Either` type. 
If the value is a `Left value` it returns `Failure value`
otherwise it returns the `Right` value wrapped in a `Success`
```
En gros, je trouve pas super lisible la syntaxe if / else dans la doc, mais j'suis pas assez dev-minded pour ça

Reply:
t'as raison de le faire remarquer et c'est effectivement pas le plus lisible.